### PR TITLE
ci: add more slo controllers e2e test

### PR DESCRIPTION
**Pull request by saintube**
### Ⅰ. Describe what this PR does

Add more E2E tests for slo-controller:

1. verify a batch pod can schedule the batch resources
2. check the cpu normalization ratio updating

Revise the deploy scripts for koord-manager and koord-scheduler.

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/webhook/elasticquota/mutating/webhooks.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
 )
 
-// +kubebuilder:webhook:path=/mutate-scheduling-sigs-k8s-io-v1alpha1-elasticquota,mutating=true,failurePolicy=fail,sideEffects=None,admissionReviewVersions=v1;v1beta1,groups=scheduling.sigs.k8s.io,resources=elasticquotas,verbs=create,versions=v1alpha1,name=melasticquota.koordinator.sh
+// +kubebuilder:webhook:path=/mutate-scheduling-sigs-k8s-io-v1alpha1-elasticquota,mutating=true,failurePolicy=ignore,sideEffects=None,admissionReviewVersions=v1;v1beta1,groups=scheduling.sigs.k8s.io,resources=elasticquotas,verbs=create,versions=v1alpha1,name=melasticquota.koordinator.sh
```
Comment (eahydra) [inline pkg/webhook/elasticquota/mutating/webhooks.go:23]: Is the FAIL strategy more appropriate?

File: test/e2e/framework/framework.go [hunk 2]
```diff
@@ -73,6 +74,7 @@ type Framework struct {
 
 	clientConfig                     *rest.Config
 	KoordinatorClientSet             koordinatorclientset.Interface
+	NRTClientSet                     nrtclientset.Interface
```
Comment (eahydra) [inline test/e2e/framework/framework.go:77]: It was previously rejected to initialize ElasticQuota's ClientSet here, but now there is an NRT Client. It is necessary to discuss whether it is necessary to initialize these ClientSets in one place, or to write some helper functions and call them in the E2E Case as needed?

File: test/e2e/framework/test_context.go [hunk 3]
```diff
@@ -335,6 +337,7 @@ func RegisterCommonFlags(flags *flag.FlagSet) {
 	// koordinator configs
 	flags.StringVar(&TestContext.KoordinatorComponentNamespace, "koordinator-component-namespace", "koordinator-system", "The namespace of the koordinator components deployed to.")
 	flags.StringVar(&TestContext.SLOCtrlConfigMap, "slo-config-name", "slo-controller-config", "The name of the slo-controller configmap.")
+	flags.StringVar(&TestContext.KoordSchedulerName, "koord-scheduler-name", "default-scheduler", "The SchedulerName of the koord-scheduler.")
```
Comment (eahydra) [inline test/e2e/framework/test_context.go:340]: The schedulerName we reveal to the outside world has always been koord-scheduler. So I recommend keeping the default value of koord-scheduler. In addition, some cases in test/e2e/scheduling have koord-scheduler written in them. Can they be corrected together?

Comment (saintube) [reply test/e2e/framework/test_context.go]: fixed

File: test/e2e/framework/framework.go (hunk 2 above)
Comment (saintube) [reply test/e2e/framework/framework.go]: fixed

**Review by eahydra — APPROVED** [review]
/lgtm
/approve

**Pull request merged**